java/src/org/openqa/selenium/os/CommandLine.java

```diff
@@ -1,174 +0,0 @@
-// Licensed to the Software Freedom Conservancy (SFC) under one
-// or more contributor license agreements.  See the NOTICE file
-// distributed with this work for additional information
-// regarding copyright ownership.  The SFC licenses this file
-// to you under the Apache License, Version 2.0 (the
-// "License"); you may not use this file except in compliance
-// with the License.  You may obtain a copy of the License at
-//
-//   http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing,
-// software distributed under the License is distributed on an
-// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
-// KIND, either express or implied.  See the License for the
-// specific language governing permissions and limitations
-// under the License.
-
-package org.openqa.selenium.os;
-
-import static org.openqa.selenium.Platform.MAC;
-import static org.openqa.selenium.Platform.WINDOWS;
-
-import java.io.File;
-import java.io.OutputStream;
-import java.util.Map;
-import java.util.concurrent.TimeUnit;
-import org.openqa.selenium.Platform;
-import org.openqa.selenium.WebDriverException;
-
-public class CommandLine {
```

Comment:
Why are we removing this class entirely? I see this is a public class and we export it as a Maven dependency https://github.com/SeleniumHQ/selenium/blob/bceee4e33098ed063dbadf8e94c881993bed0ccd/java/src/org/openqa/selenium/os/BUILD.bazel#L8. Though I don't know how many users use this. I am afraid this will be a breaking change. If we want to remove it, we can do it how we typically do it in Selenium, by deprecating it and warning the users for 2 versions at least before removing it.

Reply:
I missed this point and will rework this point to have a none breaking change.
The old classes will be deprecated and i will create a new one e.g. `ExternalProcess`.

Reply:
If we had a Project Manager, we'd have planned this out 2 releases ago so we could have done it all at the same time. 😆

Reply:
@pujagani this makes the PR smaller and better to review too, thanks for the hint.